src/utils/errorMsg.js

```diff
@@ -0,0 +1,54 @@
+// parameters
```

Comment:
you can separate them to 
```js
exports.params = {
  EMAIL: 'email',
  PASSWORD: 'password',
  FIRSTNAME: 'firstname',
  ...
};

exports.messages. = {
  EMAIL_REGISTERED: 'The email provided is already registered.',
  EMAIL_NOT_REGISTERED: 'The email provided is not registered.'
  ... 
};
```

In this way, when using these constants, we can do it like `errorMsg.params.EMAIL` or `errorMsg.messages.EMAIL_REGISTERED`, which is clearer than `errorMsg.EMAIL` or `errorMsg.EMAIL_REGISTERED`. 

Of course we can have a consensus that params will be named with only a single word, which means variables with underscores are messages. However, we cannot make sure whether a single word will be enough for all the params variables.

Reply:
changed